docs/dev/table/types.md

```diff
@@ -186,126 +152,6 @@ a table program (e.g. `field.cast(TIMESTAMP(3).bridgedTo(Timestamp.class))`) are
 </div>
 </div>
```

Comment:
I would not drop the Blink planner table yet. We still don't support all SQL DataTypes. Or we add the `Remarks for Data Type` column to each subsection.